src/components/send-money/SendMoneyAmountInput.js

```diff
@@ -80,10 +81,9 @@ class SendMoneyAmountInput extends Component {
       }
       let amountAttoNear = ''
       if (value !== '') {
-         let input = new Big(value).times(new Big(10).pow(NOMINATION))
-         amountAttoNear = input.toFixed()
+         amountAttoNear = utils.format.parseNearAmount(value);
```

Comment:
Let's replace all mentions of AttoNear to avoid confusion.

Reply:
done